src/styles/main.scss

```diff
@@ -1,3 +1,65 @@
+$gap : 101px;
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  padding: 10px;
+  margin: auto;
+  gap: 1px;
+  width: 726px;
+  flex-wrap: wrap;
```

Comment:
Please, try to follow this approach https://9elements.com/css-rule-order/.

Reply:
done